public/addreview.html

```diff
@@ -55,7 +55,7 @@ <h3 class="display-8">My Movie List</h3>
       <div class="col-md-5 offset-md-3">
         <form id="addreview">
           <div class="form-group">
-            <label for="title">Movie Title:</label>
+            <label for="title">Movie ID:</label>
```

Comment:
Should this be `title`?

Reply:
Its only the front end display text but yes it should. Should I change no or after the PR?

Reply:
Change now please, you should not have broken code (even if it's frontend display text) merged into main